api/cloudprotocol/cloudprotocol.go

```diff
@@ -192,33 +197,24 @@ type QueueInfo struct {
 
 // InstanceFilter instance filter structure.
 type InstanceFilter struct {
-	ServiceID string  `json:"serviceId"`
+	ServiceID string  `json:"serviceId,omitempty"`
 	SubjectID *string `json:"subjectId,omitempty"`
 	Instance  *uint64 `json:"instance,omitempty"`
 }
 
-// RequestServiceCrashLog request service crash log message.
-type RequestServiceCrashLog struct {
+// LogFilter request log message.
+type LogFilter struct {
+	From    *time.Time `json:"from"`
+	Till    *time.Time `json:"till"`
+	NodeIDs *[]string  `json:"nodeIds"`
```

Comment:
Here could be just slice. If it is nil, then no node ids are requested.

Reply:
Done